app/views/admin/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1>Requests</h1>
+
+<div class="admin-requests">
+  <% @requests.each do |origin, collection| %>
+    <% uniq_locations = collection.map { |h| h['origin_location'] }.uniq %>
+
+    <h3><%= SULRequests::Application.config.libraries[origin] %></h3>
+    <h4>
+      Library=<%= origin %>
+      <% if uniq_locations.size == 1 %>
```

Comment:
In rails you can use `uniq_location.one?` as a slightly shorter and more readable alternative.